src/npc/castorWildsStatue.c

```diff
@@ -15,6 +15,13 @@ extern s8 gUnk_08110E64[];
 
 extern u32 gUnk_08110E68[];
 
+typedef struct {
+    s8 x;
+    s8 y;
+} PACKED PosOffset;
+extern PosOffset gUnk_08110E78[];
```

Comment:
```suggestion
extern s8 gUnk_08110E78[];
```
`PACKED` in this context suggests that the data is really a an array accessed like arr[x * 2] arr[y * 2 + 1]

Reply:
If it's local to the file, and it matches, I'd prefer the packed struct approach - but if that change helps make it matching, we'll have to go with that ofc.

Reply:
It does not match in all cases so it isn't equivalent unfortunately.

But if it's not the source of the matching issue here and it's not referenced in other situations then we can use that.